test/onnx/test_pytorch_onnx_onnxruntime.py

```diff
@@ -11748,26 +11748,104 @@ def test_quantized_conv2d_relu(self):
         q_input = torch.quantize_per_tensor(input, 0.5, 128, torch.quint8)
         self.run_test(model, q_input)
 
+    @common_utils.parametrize(
+        "function",
+        [
+            common_utils.subtest(
+                torch.nn.ReLU(),
+                name="relu",
+            ),
+            # common_utils.subtest(
```

Comment:
Is this still in progress? Perhaps better to either skip with reason or remove from this PR.

Reply:
It is a WIP for preliminary reviews. There are four tests I needed to fix

Reply:
Done